<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>X2Focus</class>
 <widget class="QDialog" name="X2Focus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>284</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Setup USB Focus</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>12</number>
   </property>
   <item>
    <widget class="X2Form" name="X2Form">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>440</width>
       <height>260</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>440</width>
       <height>260</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QWidget" name="widget_5" native="true">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>512</y>
        <width>24</width>
        <height>24</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6"/>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>415</width>
        <height>201</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>415</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>Advanced Settings</string>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>80</y>
         <width>120</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Maximum Position:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_Normal">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>171</width>
         <height>21</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Normal Speed:  </string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_Fast">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>171</width>
         <height>21</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Fast Speed:  </string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_hs">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>110</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Half Steps:  </string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_cw">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>130</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Clockwise:  </string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_maxpos">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>80</y>
         <width>91</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>150</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Device firmware Version:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="fw_version">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>150</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>XXX</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButtonOK">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>220</y>
        <width>81</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonCancel">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>220</y>
        <width>81</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>X2Form</class>
   <extends>QFrame</extends>
   <header>x2form.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
